README.md

```diff
@@ -1 +1,33 @@
-# Webpack-Project--To-Do-List
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name: Webpack-Project: To-Do-List
+
+> Description of the project: A simple to-do-list app created using webpack.  
+
+
+## Built With
+
+- Major languages: HTML|CSS|JavaScript|Webpack
+
+
```

Comment:
- [ ] In your `README` file, there is no content for the "Getting Started" section. Kindly add content for the getting started section. This should be an example of how you may give instructions on setting up your project locally. :+1:

Reply:
Hello [mahbubul14](https://github.com/mahbubul14). Thanks for your review. However, shouldn't this be an optional suggestion? Why block my review over an omission in the README file which is not even mentioned in the project requirements? Unless I'm misunderstanding your statement.

Reply:
@TracK92, In the readme file, there is no getting started section. I think You should add a getting started section in your readme file.
You can check [this template ](https://github.com/microverseinc/readme-template)for the `README` to get an idea but remember to customize it to your project.

Reply:
@TracK92, The suggested change is a General requirement. You should always follow those general rules. It is not for a specific project. Here is the general `[requirement](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md)`.

Reply:
[mahbubul14](https://github.com/mahbubul14) noted.